tasks/offlinePackagingTasks.ts

```diff
@@ -33,8 +32,14 @@ import path = require('path');
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const argv = require('yargs').argv;
 
+interface VSIXPlatformInfo {
+    vsceTarget: string;
+    rid: string;
+    platformInfo: PlatformInformation;
+}
+
 // Mapping of vsce vsix packaging target to the RID used to build the server executable
-export const platformSpecificPackages = [
+export const platformSpecificPackages: VSIXPlatformInfo[] = [
     { vsceTarget: 'win32-x64', rid: 'win-x64', platformInfo: new PlatformInformation('win32', 'x86_64') },
     { vsceTarget: 'win32-ia32', rid: 'win-x86', platformInfo: new PlatformInformation('win32', 'x86') },
```

Comment:
Wonder how much longer we should build a win32 package. See https://code.visualstudio.com/updates/v1_84#_windows-32bit-support-ends

Reply:
Ah interesting - yeah let me see.

Reply:
It's under discussion - for now will leave this PR as-is.